transactionparser/src/channeldetails.js

```diff
@@ -0,0 +1,38 @@
+const normalizeNodes = require('./utils').normalizeNodes;
+const dropsToXRP = require('./utils').dropsToXRP;
+const BigNumber = require('bignumber.js');
+
+function summarizePaymentChannel(node) {
+
+      const final = (node.diffType === 'CreatedNode') ? node.newFields : node.finalFields;
+      const prev = node.previousFields || {};
+
+      return {
```

Comment:
What if this included a `status` (created, modified, or deleted) field similar to [parseOrderbookChanges](https://github.com/ripple/ripple-lib-extensions/tree/master/transactionparser#parseorderbookchangesmetadata)?